nikosnick13.HabitLogger/Program.cs

```diff
@@ -0,0 +1,293 @@
+
+                tableCmd.CommandText =
+                    @"CREATE TABLE IF NOT EXISTS  dring_water(
+                        Id INTEGER PRIMARY KEY AUTOINCREMENT,
+                        Date TEXT, 
+                        Quantity  INTEGER
+                    )";
+
+                tableCmd.ExecuteNonQuery();
+
+                conn.Close();
+
+
+
+            }
+
+            UserInput();
+
+            ReadKey();
+
+        }
+
+        //USER INPUT MENY
+        public static void UserInput()
+        {
+            Clear();
+            ForegroundColor = ConsoleColor.Green;
+
+            bool isAppRunning = false;
+
+            while (!isAppRunning)
+            {
+               
+                WriteLine("\n\nMAIN Menu");
+                WriteLine("\nWhat would you like to do?");
+                WriteLine("\n\tType 0 to Close Application.");
+                WriteLine("\tType 1 to Inser a Record ");
+                WriteLine("\tType 2 to View All Records");
+                WriteLine("\tType 3 to Delete a Record");
+                WriteLine("\tType 4 to Update a Record");
+                WriteLine("============================================\n");
+
+                string UserInput = ReadLine();
+
+                switch (UserInput)
+                {
+                    case "0":
+                        WriteLine("\nGoodbye!! \n");
+                        isAppRunning = true;
+                        Environment.Exit(0);
+                        break;
+                    case "1":
+                        Insert();
+                        break;
+                    case "2":
+                        VeiwAll();
+                        break;
+
+                    case "3":
+                        Delete();
+                        break;
+                    case "4":
+                        Updade();
+                        break;
+                    default:
+                        WriteLine("Invalid Command.Please type a number from 0 to 4");
+                        break;
+                }
+
+            }
+        }
+
+        #region Insert
+
+        private static void Insert()
+        {
+            Clear();
+            string date = GetDateInput();
+
+            int quantity = GetNumberInput("\n\nPlaese Insert a number of glasses or another measures of your choice (no decimals is allowed \n\n) ");
+
+            using (var conn = new SqliteConnection(connectionString))
+            {
+                conn.Open();
+
+                var insertUserCmd = conn.CreateCommand();
+                insertUserCmd.CommandText = $"INSERT INTO dring_water(date,quantity ) VALUES ('{date}',{quantity})";
+                insertUserCmd.ExecuteNonQuery();
+                conn.Close();
+            }
+
+        }
+        //Get the input for Date as string
+        private static string GetDateInput()
+        {
+            WriteLine("Please insert tha date: dd-MM-yyyy. Type 0 to return to main menu.");
```

Comment:
Typo

`WriteLine("Please insert tha date: dd-MM-yyyy. Type 0 to return to main menu.");`

`WriteLine("Please insert the date: dd-MM-yyyy. Type 0 to return to main menu.");`